app/js/containers/Issues.jsx

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { getIssues } from '../actions/issues';
+import api from '../api_wrapper';
+
+function mapStateToProps(store) {
+  return {
+    issues: store.issues
+  }
+}
+
+class IssuesPage extends React.Component {
+  constructor() {
+    super();
+  }
+
+  componentDidMount() {
+    console.log("issues mounted");
+    this.props.dispatch(getIssues());
+  }
+
+  render(){
```

Comment:
render should be somthing like 
const inssues = this.props.issues.map(issue => {
  return (
    // what ever an issue looks like
  )
})
return (
  div
    {issues}
  end div
)